events/wc_druidic_events.txt

```diff
@@ -148,7 +205,108 @@ narrative_event = {
 		}
 	}
 }
-# Suggests liege use worgens against enemies
+# Arugal returns to Gilneas, or not...
+narrative_event = {
+	id = WCDRU.304
+	title = EVTTITLE_WCDRU_304
+	desc = EVTDESC_WCDRU_304
+	picture = GFX_evt_dalaran
+	border = GFX_event_narrative_frame_war
+	
+	is_triggered_only = yes
+
+	had_character_flag = is_arugal
+	
+	trigger = {
+		OR = { 
+			top_liege = { religion = death_god }
+			k_dalaran = { has_holder = no }
+		}
+		ROOT = {
+			prisoner = no
+			is_alive = yes
+			is_landed = no
+			OR = {
+				at_location = 250
+				at_location = 251
+			}
+		}
+	}
+	
+	weight_multiplier = {
+		days = 1
+	}
+
+	option = {
+		name = EVTOPTA_WCDRU_304
+		
+		trigger = {
+			k_gilneas = { 
+				culture_group = human_group
+				NOT = { religion = death_god }
+			}
+		}
+		
+		character = ROOT
```

Comment:
This is condition that checks if 2 characters are the same
I have no idea what it's do it effect block

Reply:
Removed. The selected option was doing what it was supposed to, so I presumed this was working. Turns out it wasn't doing anything.